css/blur-no-alt-editor.css

```diff
@@ -1,32 +1,32 @@
 .block-editor-writing-flow img:is([alt=""],[alt*='.jpg'],[alt*='.png'],[alt*='.gif']) {
-	clip-path: inset(0);
-	filter: blur(8px);
-	outline: 10px solid red;
-	outline-offset: -10px;
```

Comment:
The outline is offset by a negative value of its own width to keep it inside the bounds of the image. This keeps it from interfering with adjacent elements. Is there an advantage to adding the outline to the outside?

Reply:
This was a change in a previous PR from last year. Because of the `clip-path` the outline must be inside the bounding box in order to be visible. (The `clip-path` keeps the hard edges of the image rather than having it blurred out and fading beyond its bounding box.

Reply:
I'm sorry, I'm still confused! The -10 value (as opposed to the new +10) is intended to keep the visible outline inside the bounds of the image, and not grow the image element's visible footprint. Is that not working as expected? Or am I missing some other thing?

I will admit I haven't touched this code in quite a while, so I may be missing some nuance here.

Reply:
Without the `outline-offset`, the outline is completely invisible when hovering over the image due to `clip-path`. Here's a [Codepen explaining it step-by-step](https://codepen.io/mrwweb/pen/wvYjbPx?editors=1000).

Reply:
Since these styles aren't specific to this PR, I'd recommend we "Resolve" this comment thread. If you think the styles could change, we can open up a new issue, and I'd be happy to submit a new PR once we resolve this one.